pkg/apis/kubermatic/v1/cluster.go

```diff
@@ -1010,19 +1025,19 @@ type AzureCloudSpec struct {
 	// If set to empty string at cluster creation, a new VNet will be created and this field will be updated to
 	// the generated VNet's name.
 	VNetName string `json:"vnet"`
-	// The name of a subnet in the VNet referenced by `vnet`.
+	// SubnetName is the name of a subnet in the VNet referenced by `vnet`.
```

Comment:
I see what you're doing here, following Go's "Comment should start with the field name" mantra, but IIRC this is not what we want for our CRDs, as it makes the generated documentation just look weird.

Reply:
Hey @xrstf , I got it what are you saying here so should I revert to what was there?

Reply:
Please suggest!